### ISSUE TITLE
"Blurry" texts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rakibtg/gdipp#40]
```
Which program triggers the problem? Specify the filename if possible.
Throughout the whole system

What steps will reproduce the problem?
M/A

What is the expected output? What do you see instead?
Some texts are rendered nicely while others are rendered weird

What version of gdipp are you using? On what operating system (bitness)?
0.7.5 x64, registry mode

Please provide any additional information below.
As seen in the attachment, circle #2 shows how the texts were rendered 
before (last version I tried was 0.7.3).
Circle #1 shows how some of the texts are rendered differently. This 
happens to lots of applications, for example the address bar of google 
chrome. This makes these texts harder to read.
Is it an intended change in 0.7.5? I didn't try 0.7.4 so can't comment on 
that.

```

Original issue reported on code.google.com by `ctb...@gmail.com` on 17 Apr 2010 at 4:10

Attachments:
- [Capture.JPG](https://storage.googleapis.com/google-code-attachments/gdipp/issue-40/comment-0/Capture.JPG)

**Comment by GoogleCodeExporter** [comment]
```
From what I see, the font in circle 2 is Trebuchet MS. The font in circle 1 
seems not
Segoe UI (comparing to the directory tree on the left). Do you use the default
setting? If not, please post your setting.
```

Original comment by `crendk...@gmail.com` on 17 Apr 2010 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
All I did was apply a visual style.
I've manually adjusted the font in color and appearance settings to Segoe UI 
and it 
seems to fix some of the texts. I am still having problem with the text in 
circle 1 
above and start menu (see attachment)
```

Original comment by `ctb...@gmail.com` on 17 Apr 2010 at 1:24

Attachments:
- [Untitled-1.png](https://storage.googleapis.com/google-code-attachments/gdipp/issue-40/comment-2/Untitled-1.png)

**Comment by GoogleCodeExporter** [comment]
```
Do you apply separate settings for such fonts? Can you attach your setting XML? 
Thanks.
```

Original comment by `crendk...@gmail.com` on 11 May 2010 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
Everything was default. But anyways this has been resolved in 0.7.6
```

Original comment by `ctb...@gmail.com` on 11 May 2010 at 3:43

**Comment by GoogleCodeExporter** [comment]
Original comment by `crendk...@gmail.com` on 12 May 2010 at 3:07
- Changed state: **Fixed**
